cookbook-react/.woloxci/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:8.5.0
```

Comment:
I think we don't want this `.woloxci` folder for the cookbook. Just with the husky should be enough? What do you think?

Reply:
If it works I'd keep it to be honest.